platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorStyledTextView.kt

```diff
@@ -138,14 +148,10 @@ open class EditorStyledTextView : AppCompatTextView {
     override fun onAttachedToWindow() {
         super.onAttachedToWindow()
 
-        mentionDetector = if (isInEditMode) null else newMentionDetector()
-
         updateStyle(styleConfig, mentionDisplayHandler)
     }
 
-    override fun onDetachedFromWindow() {
-        super.onDetachedFromWindow()
-
+    fun finalize() {
```

Comment:
```suggestion
    protected fun finalize() {
```

Reply:
I'm using the JNA Cleaner instead: it works quite well for this use case and It seems like finalizers are a deprecated feature in some JVMs, and although I don't think that's the case for Android, we should probably not use them much, just in case.